tests/CMakeLists.txt

```diff
@@ -328,6 +330,7 @@ add_library(test_common
         src/test_fonts/font_3.c
         unity/unity_support.c
         unity/unity.c
+        src/img_caret_down.c

```

Comment:
Agree, I can try to update the path instead.

Reply:
Actually I tried that a little bit later and because your change, by design, doesn't define `LV_BUILD_EXAMPLES` the `img_caret_down`  symbol is not defined. I wasn't testing correctly so I missed that :-(.

Reply:
Hi, thanks for your patience, the dropdown test file uses that particular asset file. I haven't worked on this since last week, will try to continue this week.